File: studio/src/main/paths.ts
import { join } from 'path'
// studio/out/main -> repo root is three levels up.
export const REPO_ROOT = join(__dirname, '..', '..', '..')
export const HIGGS = join(REPO_ROOT, 'higgs')
export const CONTENT = join(REPO_ROOT, 'content')
export const DATA = join(REPO_ROOT, 'web', 'public', 'data')

File: review/paths.py
"""Filesystem anchors for the post-review app. review/ -> repo root is one level up."""
from __future__ import annotations

import pathlib

REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent
HIGGS = REPO_ROOT / "higgs"
CONTENT = REPO_ROOT / "content"
FEEDBACK_FILE = REPO_ROOT / "feedback" / "post_comments.json"
